test/main.sh

```diff
@@ -97,6 +98,11 @@ spawn_lxd() {
 spawn_lxd 127.0.0.1:8443 $LXD_DIR
 lxd_pid=$!
 
+export LXD2_DIR=$(mktemp -d -p $(pwd))
+chmod 777 "${LXD2_DIR}"
+spawn_lxd 127.0.0.1:8444 "${LXD2_DIR}"
+lxd2_pid=$!
```

Comment:
We probably should run both LXDs on a very high port that's unlikely to be used.

My systems actually bind 8443 because I use lxd remotely, so the testsuite conflicts with my main lxd and I have to stop it.

Can we move those two to say 28443 and 28444? Ideally we'd have a function which finds us unused ports, but that may be a bit overkill for now ;)